FModel/ViewModels/GameSelectorViewModel.cs

```diff
@@ -104,7 +104,7 @@ private DetectedGame GetRiotGame(string gameName, string pakDirectory)
 
         private DetectedGame GetMojangGame(string gameName, string pakDirectory)
         {
-            _launcherSettings ??= GetDataLauncherInstalls<LauncherSettings>("\\.minecraft_dungeons\\launcher_settings.json");
+            _launcherSettings ??= GetDataLauncherInstalls<LauncherSettings>("\\.minecraft\\launcher_settings.json");
```

Comment:
I am having a hard time understanding this change

Reply:
Minecraft dungeons used to use it's own launcher, but it was merged with the normal Minecraft launcher so instead of *.minecraft_dungeons* its just *.minecraft*

Reply:
so?